src/utils/bytes.js

```diff
@@ -32,3 +32,28 @@ export function stringToBytes(str) {
   }
   return bytes;
 };
+
+/**
+ * Converts a 8-bit bytes array into a string
+ * @param {!Uint8Array} bytes
+ * @return {string}
+ */
+export function bytesToString(bytes) {
+  return String.fromCharCode.apply(String, bytes);
+};
+
+/**
+ * Generate a random bytes array with specific length
+ * @param {!number} length
+ * @return {!Uint8Array}
+ */
+export function getRandomBytesArray(length) {
```

Comment:
What I mean is the start number is random, then the pattern it generates is predictable. Contrast that with `Crypto#getRandomValues`, which has the same method signature but has no pattern (it's also a PRNG). Using `Math.random` `n` times would be better and match `Crypto`'s implementation (just without the crypto random guarantee).